docs/process/releasing-updates.md

```diff
@@ -6,25 +6,60 @@ We have three channels to which we can release: `production`, `beta`, and `test`
 
 - `production` is the channel from which the general public downloads and receives updates. It should be stable and polished.
 
-- `beta` is released more often than `production`. It may be buggy and unpolished.
+- `beta` is released more often than `production`. It may be buggy and unpolished. For example: we should be able to release at any point from the master branch.
```

Comment:
:lipstick: maybe drop the "For example" bit here. Alternative wording proposal:

> `beta` is released more often than `production`. We want to ensure `master` is always in a state where it can be released to users, and we will use `beta` for additional QA to ensure releases are polished.

Reply:
Beta